packages/test/meta.yaml

```diff
@@ -31,11 +67,18 @@ build:
     emcc ${TEST_MODULE_CFLAGS} -c Modules/_testmultiphase.c -o Modules/_testmultiphase.o
     emcc ${TEST_MODULE_CFLAGS} -c Modules/_ctypes/_ctypes_test.c -o Modules/_ctypes_test.o
 
+    for capi_src in ${TEST_CAPI_SRCS[@]}; do \
+      emcc ${TEST_MODULE_CFLAGS} -c Modules/${capi_src} -o Modules/${capi_src/.c/.o}
+    done
+
+    export TEST_CAPI_OBJECTS=( "${TEST_CAPI_SRCS[@]/#/Modules/}" )
+    emcc  ${SIDE_MODULE_LDFLAGS} ${TEST_CAPI_OBJECTS[@]//.c/.o} -o ${DISTDIR}/_testcapi.so
+
     for testname in ${TEST_EXTENSIONS}; do \
       emcc Modules/${testname/.so/.o} -o ${DISTDIR}/$testname ${SIDE_MODULE_LDFLAGS}
     done
 
     cd Lib && \
       tar --exclude=__pycache__ -cf - \
-      test distutils/tests sqlite3/test \
+      test test/testsqlite3/ \
```

Comment:
```suggestion
      test test/test_sqlite3/ \
```

I guess?

Reply:
I don't really understand why this didn't cause any failures.